modules/gdscript/tests/scripts/utils.notest.gd

```diff
@@ -24,6 +24,12 @@ static func get_type(property: Dictionary, is_return: bool = false) -> String:
 				if str(property.hint_string).is_empty():
 					return "Array[<unknown type>]"
 				return "Array[%s]" % property.hint_string
+		TYPE_DICTIONARY:
+			if property.hint == PROPERTY_HINT_DICTIONARY_TYPE:
+				if str(property.hint_string).is_empty():
+					return "Dictionary[<unknown type>, <unknown type>]"
+				var split = str(property.hint_string).split(";")
+				return "Dictionary[%s, %s]" % [split[0] if split[0] else "Variant", split[1] if split[1] else "Variant"]
```

Comment:
I would prefer to fix `hint_string` in `GDScriptParser::DataType::to_property_info()` and return to the previous version.

```suggestion
				return "Dictionary[%s]" % str(property.hint_string).replace(";", ", ")
```

Or at least use `String.get_slice()`:

```suggestion
				var key_type: String = str(property.hint_string).get_slice(";", 0)
				var value_type: String = str(property.hint_string).get_slice(";", 1)
				if key_type.is_empty():
					key_type = "Variant"
				if value_type.is_empty():
					value_type = "Variant"
				return "Dictionary[%s, %s]" % [key_type, value_type]
```